package.json

```diff
@@ -116,6 +131,9 @@
     "test:db:migrate": "NODE_ENV=tests knex migrate:latest --knexfile db/knexfile.js --env tests",
     "test:tests": "NODE_ENV=tests HIBP_THROTTLE_DELAY=1000 HIBP_THROTTLE_MAX_TRIES=3 jest --runInBand --coverage tests/",
     "test:coveralls": "cat ./coverage/lcov.info | coveralls",
+    "test:integration": "./node_modules/.bin/wdio tests/integration/wdio.conf.js",
```

Comment:
nit: Typically, npm scripts will add `./node_modules/.bin/` to the current `$PATH`, so you might be able to just call `"wdio tests/integration/wdio.conf.js"` without the prefixes (here and on L135+136 below)